src/__swaps__/utils/swaps.ts

```diff
@@ -232,7 +235,7 @@ export function trimTrailingZeros(value: string) {
 export function precisionBasedOffMagnitude(amount: number | string, isStablecoin = false): number {
   'worklet';
 
-  const magnitude = -Number(floorWorklet(sumWorklet(log10Worklet(amount), 0)));
+  const magnitude = -Number(floorWorklet(log10Worklet(amount)));
```

Comment:
Gonna conflict with brunos PR

Reply:
It's fine. I'll rebase later but I'll think we should keep my change